test/core/test_ulid_usecase.rb

```diff
@@ -28,11 +28,7 @@ def test_monotonic_generator_with_no_moment_depends_on_current_time
 
     sorted_times.each_cons(2) do |pred, succ|
       assert do
-        pred.to_i == succ.to_i
-      end
-
-      assert do
-        ULID.milliseconds_from_moment(pred) < ULID.milliseconds_from_moment(succ)
```

Comment:
This is needless. Because ensured at above

```ruby
    ulids_by_the_time = ulids.group_by(&:to_time)
    sorted_times = ulids_by_the_time.keys
```